src/styles/blocks/_basic.scss

```
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 30px;
```

```suggestion
  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
  font-size: 30px;
```
fonts should be in `html {...}` tag